docs/SelectMultiWithCustomOptions.stories.ts

```diff
@@ -0,0 +1,44 @@
+import SelectMulti from '../src/SelectMulti.vue'
+
+import { longDefaultListOfOptions } from './storybookUtilities'
+
+export default {
+	title: 'SelectMulti with custom options',
+	component: SelectMulti,
+};
+
+const Template = (args, { argTypes }) => ({
+	props: Object.keys(argTypes),
+	components: { SelectMulti },
+	data() {
+		return { values: [{}] } },
+	template: `
+	<div class="wrapper">
+		<SelectMulti
+			v-model="values"
+			:options="options"
+			:label="label"
+			:labelIsVisible="labelIsVisible"
+			:placeholder="placeholder"
+			:disabled="disabled"
```

Comment:
@RED143 - this is all looking great. I think the only thing we have to add is the ability in `SelectMulti` to pass in a custom matching function. We can't really *know* how a consumer will want to match the content in the slot, since it could involve HTML & we don't want to try to parse that. So instead, if they use a slot, they should have the option to also pass in a custom matcher. That should allow them to then make sure that any labels / values / other info they provide in the slot also works when typing.

Demo of typing in the SelectMulti that a consuming project would want to be possible:
![Screen Recording 2020-10-23 at 07 47 01 AM](https://user-images.githubusercontent.com/14101491/96999965-10bff480-1504-11eb-9eb2-9a76adff67d4.gif)

Reply:
vue-multiselect calls this `customLabel` as a prop: https://vue-multiselect.js.org/#sub-custom-option-template

Where you pass in a method that accepts an option. vue-multiselect then calls that method on each option to determine what to match against for each option, when comparing to the user's search string. Let me know if that makes sense?

Reply:
I did something similar in gamma for local version of this package:
https://stash.politico.com/projects/SVC/repos/gamma/pull-requests/3913/overview
Is it looks ok?

Reply:
@jackkoppa How it looks:
![search mov](https://user-images.githubusercontent.com/44836424/97012591-a6817100-1550-11eb-834b-126480ba2147.gif)

Guys, I don't sure about `customLabel` prop name, If you have better name for this property, please, share. (My version is `optionLabelForSearching`)

Reply:
`optionLabelForSearching` is **perfect** 🎉